R/data-baby_names.R

```diff
@@ -0,0 +1,34 @@
+#' Baby Names Data.
```

Comment:
How does this relate to https://cran.r-project.org/web/packages/babynames/index.html? Is it worth doing a PR there to update instead?

Reply:
I'll look into this!

Reply:
Looks like it should definitely be there to update instead of here.

Reply:
Might be worth opening an issue there to see if Hadley is interested in an update.

Reply:
Will do!